src/json-editor.component.html

```diff
@@ -13,7 +13,7 @@
       </tr>
       <tr>
         <td>
-          <any-type-field [value]="_record.get(key)" [schema]="schema.properties[key]" [path]="getPathForField(key)"></any-type-field>
+          <any-type-field [value]="_record.get(key) | selfOrEmpty:schema.properties[key]" [schema]="schema.properties[key]" [path]="getPathForField(key)"></any-type-field>
```

Comment:
no it was a leftover from when i was trying to implement top level fields addition with the button below tree menu.